src/v3/test/e2e/tests/theming.ts

```diff
@@ -13,7 +13,7 @@
 import { Selector } from 'testcafe';
 
 fixture('Theming')
-  .page('http://localhost:3000/?siw-use-mocks=true&siw-mock-response=/idp/idx/introspect/default');
```

Comment:
Changed to use scenario because this mock does not allow navigation to different screens (as the test is attempting to do).